homeassistant/components/mill/climate.py

```diff
@@ -52,11 +49,11 @@
 )
 
```

Comment:
(comment not related to line but to file)

For a future PR, we should register the service using `entity_platform.async_register_entity_service`